frontend/public/components/pod.jsx

```diff
@@ -185,6 +187,8 @@ export const PodDetailsList = ({pod}) => {
     <dd>{pod.status.podIP || '-'}</dd>
     <dt>Node</dt>
     <dd><NodeLink name={pod.spec.nodeName} /></dd>
+    <dt>Tolerations</dt>
+    <dd><a className="co-m-modal-link" onClick={Kebab.factory.ModifyToleration(PodModel, pod).callback}>{pluralize(_.size(pod.spec.tolerations), 'Toleration')}</a></dd>
```

Comment:
We should add this to other workload pages.

For pods, tolerations cannot be edited. So this should be a readonly modal.